src/NuGet.Clients/NuGet.PackageManagement.UI/Actions/PackageIdentityResult.cs

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Globalization;
+using NuGet.Packaging.Core;
+using NuGet.Versioning;
+
+namespace NuGet.PackageManagement.UI
+{
+    /// <summary>
+    /// Represents a package identity with an acessible name in the Preview Window
+    /// </summary>
+    public class PackageIdentityResult : PackageIdentity
```

Comment:
Not crazy about this naming convention.  What do you think about `AccessiblePackageIdentity`?

Make it `sealed`.

Reply:
I was trying to follow the pattern name in `PackageIdentityResult` and [`UpdatePreviewResult`](https://github.com/NuGet/NuGet.Client/blob/319fa107e1c90be67da6e1f57f80de9398dfdc3c/src/NuGet.Clients/NuGet.PackageManagement.UI/Actions/UpdatePreviewResult.cs#L8). The latter also creates an acessible name for a package update in the Preview Windows in the UI. I will change `PackageIdentityResult` to `sealed` and rename it.